src/main/java/ru/portfolio/portfolio/parser/ReportRestController.java

```diff
@@ -106,7 +106,7 @@
                                 e.printStackTrace(pw);
                                 return sw.toString().replace("\n", "</br>");
                             }).collect(Collectors.joining("</br></br> - ",
-                                    "<b>Ошибка загрузки отчетов</b></br></br> - ",
+                                    "<b>Ошибка загрузки отчетов</b> <a href=\"/\">[ok]</a></br></br> - ",
```

Comment:
Предлагаю назад, т.к. тут ситуация не ok. Можно предложить помочь в решении ошибки.
```suggestion
                                    "<b>Ошибка загрузки отчетов</b> <a href=\"/\">[назад]</a><br/>" +
                                    "<span style=\"font-size: smaller; color: gray;\">Вы можете " +
                                    "<a href=\"https://github.com/vananiev/portfolio/issues\">сообщить</a> об ошибке " +
                                    "разработчикам</span></br></br> - ",
```